cases/ts-runtime-checks/src/index.ts

```diff
@@ -0,0 +1,17 @@
+import type { Assert } from 'ts-runtime-checks';
+
+interface ToBeChecked {
+  number: number;
+  negNumber: number;
+  maxNumber: number;
+  string: string;
+  longString: string;
+  boolean: boolean;
+  deeplyNested: {
+    foo: string;
+    num: number;
+    bool: boolean;
+  };
+}
+
+export const assertLoose = (value: Assert<ToBeChecked>) => true;
```

Comment:
Please add a comment to silence the eslint warning.

Reply:
added ;)